backup-manager/src/main/java/com/openmc/backupmanager/service/BackupService.java

```diff
@@ -0,0 +1,184 @@
+package com.openmc.backupmanager.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+@Service
+public class BackupService {
+
+    private static final Logger logger = LoggerFactory.getLogger(BackupService.class);
```

Comment:
Suggestion (non-blocking): Use Lombok annotation instead of instantiating a logger manually